lib/web/assistant.go

```diff
@@ -284,9 +295,25 @@ func runAssistant(h *Handler, w http.ResponseWriter, r *http.Request, sctx *Sess
 			return trace.Wrap(err)
 		}
 
-		if err := processComplete(h, r.Context(), chat, conversationID, ws, authClient); err != nil {
+		completionTokens, err := processComplete(h, r.Context(), chat, conversationID, ws, authClient)
+		if err != nil {
 			return trace.Wrap(err)
 		}
+
+		usageEventReq := &proto.SubmitUsageEventRequest{
+			Event: &usageeventsv1.UsageEventOneOf{
+				Event: &usageeventsv1.UsageEventOneOf_AssistCompletion{
+					AssistCompletion: &usageeventsv1.AssistCompletionEvent{
+						TotalTokens:      int64(len(promptTokens) + completionTokens),
+						PromptTokens:     int64(len(promptTokens)),
+						CompletionTokens: int64(completionTokens),
+					},
+				},
+			},
+		}
+		if err := authClient.SubmitUsageEvent(r.Context(), usageEventReq); err != nil {
+			h.log.WithError(err).Error("Failed to emit usage event")
```

Comment:
Can you please change the log level to waning or debug? This is not a critical error, and this will print ERROR message in our logs and the whole call stack. It's hard to debug errors if non-critical logs are marked as errors. Our users are also very often confused by that.

Reply:
+1 for warning, we don't even log at error level if we fail to emit audit log events, most of the time.